### DISCUSSION TITLE
Blog landing page with optionals: banner/image, profile pic, short bio, social links, latest blog posts

### DISCUSSION THREAD

**Discussion by fredrikaverpil** [squidfunk/mkdocs-material#5278] [Ideas]
I'm using mkdocs-material for technical documentation at work but also for my personal blog (the insiders blog feature). I love this, and the only thing I kind of miss is a pretty landing page.

Some free blog themes out there employ the option to set up a landing page with e.g. a banner image, a profile picture, some social links and a brief introduction to yourself and/or your blog. Then you usually see a list of the most recent blog entries.

I'm looking in awe at the [mkdocs-material landing page](https://squidfunk.github.io/mkdocs-material/), but at the same time realize I don't have the time/ambition to set something unique like that up. I'd rather have the option to use a default but nice landing page which is focussed around the fact that I'm primarily running a technical blog. Right now I've simply created the good old fashioned "about me" page, which I've been employing since the 90's.

Thoughts?

**Comment by squidfunk** [comment]
Thanks for your feedback! Yes, our own landing page was quite some work to pull off. You can set up a landing page for your blog like you would for your documentation. Since this is a customization, please see the following issues for guidance:

- #1996
- #2057

We'll add new landing page templates and more blog index templates in the future but it's currently not a priority given how many other stuff there is to do on the functional side. We'll try our best to offer more blog functionalities in the future, but please keep in mind that this project is still maintained by a solo developer 😅 Hopefully, with increasing funding, this might change in the future.

---

__Question__: How do you configure landing page templates on the other blog themes / generators you mentioned? Input would be great, because I'm pretty sure it will always involve customization (over configurarion).
